index.html

```diff
@@ -13,10 +13,10 @@
     <script type="text/javascript" class="remove">
       var respecConfig = {
         // specification status (e.g. WD, LCWD, NOTE, etc.). If in doubt use ED.
-        specStatus: "CG-FINAL",
+        specStatus: "FPWD",
 
         // the specification's short name, as in http://www.w3.org/TR/short-name/
-        shortName: "claims-data-model",
+        shortName: "verifiable-claims-data-model",
```

Comment:
What I inherited for use cases was "verifiable-claims-use-cases", which is what I believe @liamquin applied for. While I can see that this may not be "short", there is some reason to want to use the same prefix for both documents. Not too late to change use cases, but probably leads to a short delay.